default.nix

```diff
@@ -0,0 +1,117 @@
+{ stdenv
+, poetry2nix
+, file
+, uwsgi
+, ghostscript
+, pkgs
+, lib
+, runCommand
+, python38
+, writeScriptBin
+, version ? "git"
+}:
+let
+  concrexit-python = python38;
+
+  src = ./.;
+
+  poetryPython = poetry2nix.mkPoetryPackages {
+    projectDir = src;
+    overrides = poetry2nix.overrides.withDefaults (
+      _self: super: {
+        uwsgi = {};
+        python-magic = concrexit-python.pkgs.python_magic;
+        cryptography = super.cryptography.overridePythonAttrs (old: {
+          cargoDeps = pkgs.rustPlatform.fetchCargoTarball {
+            inherit (old) src;
+            name = "${old.pname}-${old.version}";
+            sourceRoot = "${old.pname}-${old.version}/src/rust/";
+            sha256 = "sha256-tQoQfo+TAoqAea86YFxyj/LNQCiViu5ij/3wj7ZnYLI=";
+          };
+          cargoRoot = "src/rust";
+          buildInputs = old.buildInputs ++ [ pkgs.libiconv ];
+          nativeBuildInputs = old.nativeBuildInputs ++ (with pkgs.rustPlatform; [
+            rust.rustc
+            rust.cargo
+            cargoSetupHook
+          ]);
+        });
+      }
+    );
+    python = concrexit-python;
+  };
+
+  inherit (poetryPython) poetryPackages pyProject;
+
+  concrexit-env = poetryPython.python.buildEnv.override { extraLibs = poetryPackages; };
+
+  # We want to enable the python3 plugin for uwsgi, so we override here
+  uwsgi-python = (uwsgi.override { plugins = [ "python3" ]; python3 = concrexit-python; }).overrideAttrs (old: { buildInputs = old.buildInputs ++ [ pkgs.zlib pkgs.expat ]; });
+
+  # Wrapper script that sets the right options for uWSGI
+  concrexit-uwsgi = writeScriptBin "concrexit-uwsgi" ''
+    #! ${pkgs.runtimeShell}
+    cd ${concrexit}/website
+    MANAGE_PY=1 ${concrexit-env}/bin/python ${concrexit}/website/manage.py migrate
```

Comment:
does this maybe also require `manage.py collectstatic` and `manage.py compress`

Reply:
This is done during built time right here: https://github.com/svthalia/concrexit/blob/0e922a106970c6d6a242979a70554a5b6676b594/default.nix#L89-L90